--- pyeemd/pyeemd.py	2017-06-14 16:26:14.537861539 +1000
+++ pyeemd/pyeemd-update.py	2017-06-14 16:29:57.396048678 +1000
@@ -41,18 +41,7 @@
 # Load libeemd.so

 def _init():
-    # First try 'libeemd.so' in current directory
-    dirname = os.path.dirname(os.path.realpath(__file__))
-    check_first_names = ["libeemd.so"]
-    for libfile in [os.path.join(dirname, filename) for filename in check_first_names]:
-        if os.path.exists(libfile):
-            return ctypes.CDLL(libfile)
-    # Then try find_library
-    lib = find_library("eemd")
-    if lib:
-        return ctypes.CDLL(lib)
-    else:
-        raise RuntimeError("Cannot find libeemd C library. Tried directory '%s' and ctypes.util.find_library" % dirname)
+    return ctypes.CDLL("libeemd.so")

 _libeemd = _init()
